api/controller/pokemon_controller.go

```diff
@@ -0,0 +1,36 @@
+package controller
+
+import (
+	"go-bootcamp/service"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"github.com/pkg/errors"
+)
+
+type PokemonController interface {
```

Comment:
All the exported stuff (interface, struct, function, etc) should have a description, you can use this template
```go
// PokemonController contains the methods for...
```

Reply:
comments added to the required components